### ISSUE TITLE
Steam Game Menu - Properties not working

### ISSUE THREAD

**Issue by Pythonic456** [ptitSeb/box86#295]
https://user-images.githubusercontent.com/56113118/103443247-29abfb00-4c55-11eb-9760-5ac10b4140d9.mp4

This doesn't work for any other game, it doesn't crash or output any errors or anything abnormal.

Thanks :)

**Comment by ptitSeb** [comment]
There is no error message on the console. It maybe that this is just not supported anymore (keep in mind Steam on 32bits OS is not supported by Steam anymore).

**Comment by stormchaser3000** [comment]
I noticed that the uninstall button doesn't work either... but that might be a separate issue. I tested steam on my AMD64 machine again and it looks like, from a strictly visual observation, steam appears to use it's new web browser user interface for the properties page. That could be why it isn't working.

On that note, does anyone know how to manually change the runtime properties of a steam game by editing a file under .local/share/Steam?
